lib/AST/AvailabilityConstraint.cpp

```diff
@@ -124,7 +124,7 @@ static bool canIgnoreConstraintInUnavailableContexts(
     // which they are available. However, make an exception for types and
     // conformances, which can sometimes be awkward to avoid references to.
     if (!isa<TypeDecl>(decl) && !isa<ExtensionDecl>(decl)) {
-      if (domain.isUniversal() || domain.isSwiftLanguage())
+      if (domain.isUniversal() || domain.isSwiftLanguage() || domain.isSwiftToolchain())
```

Comment:
Since I don't think availability in the `_SwiftToolchain` domain should be diagnosed (in this release, at least) I don't think this should be updated.